### ISSUE TITLE
IdentityRecentActivity should be applicable for all users

### ISSUE THREAD

**Issue by oxisto** [clouditor/clouditor#1552]
Currently, it has only privileged users. Should be applicable for all users

https://github.com/clouditor/clouditor/blob/8f3953f3088308d7fb37292c45d25e363788be4e/policies/bundles/IdentityManagement/IdentityRecentActivity/metric.rego#L13

Also we should possibly rename it?

**Comment by anatheka** [comment]
Yes, we should delete l13 and rename it. IdentityTimeSinceLastActivity is better to understand.

**Comment by anatheka** [comment]
Already fixed in the new metrics repo. A PR for renaming is open in the new metrics repo.
